terraform/app/modules/paas/variables.tf

```diff
@@ -69,4 +70,5 @@ locals {
   logging_service_name     = "${var.service_name}-logit-${var.environment}"
   postgres_service_name    = "${var.service_name}-postgres-${var.environment}"
   web_app_name             = "${var.service_name}-${var.environment}"
+  csv_bucket_name          = "${var.service_name}-csv-${var.environment}"
```

Comment:
Maybe include `bucket` in the name - all other services include the answer to 'what are you?' in their names.